src/lib/PickerComponent.ios.js

```diff
@@ -29,26 +25,27 @@ class RenderedSelector extends React.Component {
   }
 
   render() {
+    
     let picker = (
-      <TPicker
-        tid="Picker"
+      <Picker
+        testID={`Picker`}
         {...this.props.pickerProps}
         selectedValue={this.state.value}
         onValueChange={this.handleValueChange.bind(this)}
         mode="dropdown"
       >
         {this.props.options.map(
           ({ value, label }, idx) => (
-            <TPickerItem
-              tid={`PickerItem[${idx}]`}
+            <Picker.Item
+                testID={`PickerItem/${idx}`}
```

Comment:
Is there some formatting issues here with the extra indentation?  Is prettier running?